Analysis/bin/ShuffleMergeSpectral.cxx

```diff
@@ -46,30 +48,30 @@ struct Arguments {
     run::Argument<double> start_entry{"start-entry", "starting ratio from which file will be processed", 0};
     run::Argument<double> end_entry{"end-entry", "end ratio until which file will be processed", 1};
     run::Argument<double> exp_disbalance{"exp-disbalance", "maximal expected disbalance between low pt and high pt regions",0};
+    run::Argument<unsigned> compression_level{"compression-level", "compression level of output file", 4};
```

Comment:
For completeness, the compression algorithm should also be passed as an argument.